improver/cli/combine.py

```diff
@@ -86,8 +84,6 @@ def process(
         )
 
     else:
-        result = CubeCombiner(operation)(
-            CubeList(cubes), new_name, use_midpoint=use_midpoint,
-        )
+        result = CubeCombiner(operation)(CubeList(cubes), new_name)
```

Comment:
Sorry this isn't related to your changes but I notice that the `check_metadata` argument in this CLI is unused. Could you remove this argument and associated docstring?

Reply:
Done. I also checked no apps in the suite are providing this redundant option; they aren't.